lib/vagrant-libvirt/config.rb

```diff
@@ -502,7 +502,11 @@ def finalize!
         @memory = 512 if @memory == UNSET_VALUE
         @cpus = 1 if @cpus == UNSET_VALUE
         @cpu_mode = 'host-model' if @cpu_mode == UNSET_VALUE
-        @cpu_model = 'qemu64' if @cpu_model == UNSET_VALUE
+        @cpu_model = if (@cpu_model == UNSET_VALUE and @cpu_mode == 'custom')
+            'qemu64'
```

Comment:
That's the point of this PR ;-)

While it is okay to set "random" values for `cpu_model` when `cpu_mode` is `custom`, but when set to `host-model` or `host-passthrough` it is not really supported (https://libvirt.org/formatdomain.html#elementsCPU) and may even make the VM fail to start (https://www.redhat.com/archives/libvir-list/2016-May/msg01940.html).

Thus the idea was to to keep the `qemu64` model while in custom mode, but reset it to an empty string in any other mode.